internal/util/path.go

```diff
@@ -297,3 +297,13 @@ func ShouldNormalizePath(path string) string {
 
 	return realPath
 }
+
+// Best effort attempt to determine the absolute path to the directory that holds a custom package path.
+// The package path is expected to be an internal path of the form `github.com/edulinq/autograder/`.
+func getDirPathFromCustomPackagePath(packagePath string) string {
+	if strings.HasPrefix(packagePath, "github.com/edulinq/autograder/") {
+		packagePath = strings.TrimPrefix(packagePath, "github.com/edulinq/autograder/")
+	}
+
+	return ShouldAbs(filepath.Join(ShouldGetThisDir(), "..", "..", packagePath))
```

Comment:
There is a bit of a mix of path types here.
I think the package path in Go has to have forward slashes, so that's fine.
But, then it could still have those forward slashes when you join.
I know we are POSIX only, but we try to be flexible as we can.